crates/c/src/lib.rs

```diff
@@ -1725,7 +1733,7 @@ impl InterfaceGenerator<'_> {
         if abi::guest_export_needs_post_return(self.resolve, func) {
             uwriteln!(
                 self.src.c_fns,
-                "__attribute__((__weak__, __export_name__(\"cabi_post_{export_name}\")))"
+                "__attribute__((__used__, __export_name__(\"cabi_post_{export_name}\")))"
```

Comment:
Can this be both "used" and "weak"? The intention for weak IIRC is to enable overriding this symbol if the default behavior isn't desired

Reply:
Ah, those were left over from an experiment I was doing but ended up reverting. I've now removed the `__used__`s too.